python/tk_desktop/desktop_window.py

```diff
@@ -504,9 +503,11 @@ def show_update_project_config(self):
         self.update_project_config_widget.show()
         self.project_overlay.hide()
 
+    def _get_current_user(self):
+        return login.get_current_user(sgtk.platform.current_engine().tank)
```

Comment:
I attached a comment to the commit to that effect, but it should be in the code as well. Will do.